selenium/pageObject_model/HomePageSitePastebin.js

```diff
@@ -0,0 +1,131 @@
+import pkg from 'selenium-webdriver';
+const { Builder, Browser, By, until, wait } = pkg;
+import { Basic } from './Basic.js';
+
+export class HomePageSitePastebin extends Basic {
+    constructor(driver) {
+        super(driver);
+        this.urlHomePastebin = 'https://pastebin.com';
+        this.formXpath = '//*[@id="w0"]';
+        this.textareaXpath = '//*[@id="postform-text"]';
+        this.buttonCloseBannerXpath = '//*[@id="hideSlideBanner"]';
+        this.syntaxSelectXpath = '//*[@id="select2-postform-format-container"]';
+        this.expirationSelectXpath =
+            '//*[@id="select2-postform-expiration-container"]';
+        this.syntaxListXpath =
+            '//*[@id="select2-postform-format-results"]/li[2]/ul';
+        this.expirationListXpath =
+            '//*[@id="select2-postform-expiration-results"]';
+        this.syntaxItemXpath = '//li[contains(text(), "Bash")]';
+        this.expirationItemXpath = '//li[contains(text(), "10 Minutes")]';
+        this.pasteTitleXpath = '//*[@id="postform-name"]';
+        this.buttonCreateNewPasteXpath =
+            '//*[@class="btn -big" and @type="submit"]';
+        this.form = null;
+        this.fieldOfPaste = null;
+        this.buttonCreateNewPaste = null;
+        this.pasteTitle = null;
+        this.frameBanner = null;
+        this.syntaxSelect = null;
+        this.syntaxList = null;
+        this.syntaxItem = null;
+        this.expirationSelect = null;
+        this.expirationItem = null;
+        this.expirationList = null;
+        this.pasteTitleText = null;
+        this.buttonCloseBanner = null;
+        this.dataTitleForTaskOne = 'helloweb';
+        this.dataTitleForTaskTwo = 'how to gain dominance among developers';
+        this.dataTextareaForTaskOne = 'Код: "Hello from WebDriver"';
+        this.dataTextareaForTaskTwo =
+            'git config --global user.name  "New Sheriff in Town"' +
+            '\n' +
+            'git reset $(git commit-tree HEAD^{tree} -m "Legacy code")' +
+            '\n' +
+            'git push origin master --force';
```

Comment:
I left only the entered data with + "/ n", because I need to store this data somewhere, in which I then need to remove the spaces and convert to lower case (for comparison) and need to store this variable to be used in the mocha unit framework